karbon/karbon_cluster_registration_service_test.go

```diff
@@ -118,6 +115,81 @@ func TestKarbonCreateClusterRegistration(t *testing.T) {
 	// TODO get task uuid status
 }
 
+func TestKarbonCreateClusterRegistrationWithNoCategory(t *testing.T) {
+	interceptor := khttpclient.NewInterceptor(http.DefaultTransport)
+	creds := testhelpers.CredentialsFromEnvironment(t)
+	nkeClient, err := NewKarbonAPIClient(creds, WithRoundTripper(interceptor))
+	require.NoError(t, err)
+
+	kctx := mock.NewContext(mock.Config{
+		Mode: keploy.MODE_TEST,
+		Name: t.Name(),
+	})
+
+	test_cluster_name := strings.ToLower("cluster2")
+	test_cluster_uuid := strings.ToLower("E33FD0FD-5673-45FA-825D-7EF869A91577")
+	test_metadata_apiversion := "v1.1.0"
+	test_metadata := &Metadata{APIVersion: &test_metadata_apiversion}
+
+	t.Log("Get Cluster registration")
```

Comment:
I'd advise against using `t.Log` in a unit test scenario. These make some sense in long-running integration tests but for unit-tests your test debugging guide should be machine runnable assertions rather than human readable logs.